src/utility/HCITransport.h

```diff
@@ -22,6 +22,13 @@
 
 #include <Arduino.h>
 
+typedef enum BLEChip_s {
+  SPBTLE_RF,
```

Comment:
Why keeping type of controllers that are not supported anyway by the driver  ?

Reply:
This enum is for defining the BLEChip_s of different boards.
so it's mentionning the RF part of the stm32WB